src/state/reducers/animationReducer.js

```diff
@@ -1,13 +1,18 @@
 const initialState = {
-  sidebarVisible: false
+  sidebarVisible: false,
+  legendVisible: false,
 }
 
 const locationReducer = (state = initialState, action) => {
   switch (action.type) {
```

Comment:
can we change that name to animationReducer ? location is irrelevant.

Reply:
yes